ParcelDistributionCenter/ParcelDistributionCenter.Web/Program.cs

```diff
@@ -10,7 +10,7 @@ public static void Main(string[] args)
 
             // Add services to the container.
             builder.Services.AddControllersWithViews();
-            builder.Services.AddScoped<MemoryRepository>();
+            builder.Services.AddScoped<IMemoryRepository, MemoryRepository>();
```

Comment:
Aaa tu widzę dodałeś z powrotem xd

Reply:
Tak dodałem bo to się powinno nam generalnie przydać.